index.js

```diff
@@ -1,6 +1,28 @@
 var React = require('react');
+var ReactDOM = require('react-dom');
 var assign = require('object-assign');
 
+var hasConsole = typeof window !== 'undefined' && window.console;
+var noop = function() {}
+var swallowInvalidHeadWarning = noop;
+var resetWarnings = noop;
+
+if(hasConsole) {
+  var originalError = console.error;
+  // Rendering a <head> into a body is technically invalid although it
+  // works. We swallow React's validateDOMNesting warning if that is the
+  // message to avoid confusion
+  swallowInvalidHeadWarning = function() {
```

Comment:
We'll need to find a better solution to this but for now this is ok I'll create a follow up issue to explore alternatives.